packages/extension-driver-snowflake/src/lib/snowflakeDataSource.ts

```diff
@@ -100,6 +97,78 @@ export class SnowflakeDataSource extends DataSource<any, SnowflakeOptions> {
     }
   }
 
+  public override async export(exportOptions: ExportOptions): Promise<void> {
+    let { directory } = exportOptions;
+    const { profileName, sql } = exportOptions;
+
+    const { connection, pool } = await this.getConnection(profileName);
+    directory = /^\./.exec(directory) ? directory : `./${directory}`;
+
+    // throw error if filepath dir not exist
+    if (!fs.existsSync(directory)) {
+      throw new InternalError(`Directory ${directory} not exist`);
+    }
+
+    try {
+      const stage = '@~/vulcan_cache_stage';
+      const stageFilePath = `${stage}/`;
+      const builtSQL = this.getCopyToStageSQL(sql, stageFilePath);
+      // copy data to a named stage
```

Comment:
Suggest to rich the comment to explain why we need to copy to the stage to help members know moving to the stage is necessary.